rai/node/common.hpp

```diff
@@ -313,6 +328,77 @@ class node_id_handshake : public message
 	static size_t constexpr query_flag = 0;
 	static size_t constexpr response_flag = 1;
 };
+class musig_stage0_req : public message
+{
+public:
+	musig_stage0_req (bool &, rai::stream &, rai::message_header const &);
```

Comment:
Any reason we've stopped using parameter names in declarations? The class declaration is the interface you look at first, and dropping param names makes for a hard read IMO (and doxygen doesn't pick them up from cpp files)

Reply:
Yeah IDK... My editor (VSCode) doesn't pick them up from the C++ file either which is a real pain.